.circleci/config.yml

```diff
@@ -134,7 +133,6 @@ jobs:
         paths:
           - build/_output
           - pkg/apis/db/v1alpha1/zz_generated.deepcopy.go
-          - pkg/apis/db/v1alpha1/zz_generated.deepcopy.go
```

Comment:
We need to cache the other generated file

Reply:
Why ?

Reply:
We don’t need to cache the same file twice and there is only one generated file. Default is not there anymore